alf/trainers/policy_trainer.py

```diff
@@ -200,6 +255,9 @@ def _markdownify(paragraph):
             env_steps_metric = self._algorithm.get_step_metrics()[1]
             total_time_steps = env_steps_metric.result()
             iter_num += 1
+
+            self._trainer_progress.update(iter_num, total_time_steps)
+
             if ((self._num_iterations and iter_num >= self._num_iterations)
                     or (self._num_env_steps
                         and total_time_steps >= self._num_env_steps)):
```

Comment:
Can use ```progress()```  to simplify Line 261-263

Reply:
Might have floating error when the integer number is huge